lib/html/pipeline.rb

```diff
@@ -42,6 +42,8 @@ class Pipeline
     autoload :TextileFilter,         'html/pipeline/textile_filter'
     autoload :TableOfContentsFilter, 'html/pipeline/toc_filter'
     autoload :TextFilter,            'html/pipeline/text_filter'
+    
+    MissingDependencyError = Class.new(LoadError)
```

Comment:
Could you switch this to this syntax:

```ruby
class MissingDependencyError < LoadError; end
```

It's a bit easier for ctags, and editors to find the definition this way.